packages/schema/src/types/discriminatedObject.ts

```diff
@@ -39,43 +45,64 @@ export function discriminatedObject<
         return discriminatorMap[discriminatorValue];
       }
     }
+    return undefined;
+  };
+  const allSchemas = Object.values(discriminatorMap).reverse();
+  const selectSchema = (
+    value: unknown,
+    discriminatorProp: string | TDiscrimProp | TDiscrimMappedProp,
+    checker: (schema: TSchema) => SchemaValidationError[],
+    isAttr: boolean = false
```

Comment:
does the default value "false" has any impact on the code below?

Reply:
You are it doesn't have any impact, removing it^